namerd/storage/k8s/src/main/scala/io/buoyant/namerd/storage/experimental/kubernetes/resources.scala

```diff
@@ -0,0 +1,28 @@
+package io.buoyant.namerd.storage.experimental.kubernetes
+
+import io.buoyant.k8s.{Client, NsThirdPartyVersion, ThirdPartyVersion}
+
+case class Api(client: Client) extends ThirdPartyVersion[Dtab] {
+  override def owner: String = Api.Owner
+
+  override def ownerVersion: String = Api.OwnerVersion
+
+  override def withNamespace(ns: String) = new NsApi(client, ns)
+  implicit val descriptor = DtabDescriptor
+  def dtabs = listResource[Dtab, DtabWatch, DtabList]()
+}
+
+object Api {
+  // TODO: should this be buoyant.io, linkerd.io, namerd.io...?
+  val Owner = "l5d.io"
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Field name 'Owner' does not match the regular expression '^[a-z][A-Za-z0-9]*$'](https://www.codacy.com/app/ver/linkerd/file/2415518826/issues/source?bid=3236810&fileBranchId=3236828#l17)